plugins/modules/zos_copy.py

```diff
@@ -2031,7 +2039,7 @@ def main():
         validate=dict(arg_type='bool', required=False),
         sftp_port=dict(arg_type='int', required=False),
         volume=dict(arg_type='str', required=False),
-
+        tmphlq=dict(arg_type='qualifier_or_empty', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.

Reply:
Resolving because I created issue https://github.com/ansible-collections/ibm_zos_core/issues/387 - it was a good comment and point, please review the issue as well @fernandofloresg , also see the other work item that came from this PRs review https://github.com/ansible-collections/ibm_zos_core/issues/386